metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -736,6 +751,19 @@ private TableInfo getFromTableServiceProxy(final QualifiedName name,
                 : connectorTableServiceProxy.get(name, getTableServiceParameters, useCache);
     }
 
+    private void deleteCommonViewStorageTable(final QualifiedName viewName,
+                                              final QualifiedName storageTableName) {
+
+        try {
+            Log.warn("Deleting storage table: {} belonging to common view: {}",
+                    storageTableName, viewName);
+            connectorTableServiceProxy.delete(storageTableName);
```

Comment:
This should be calling method `TableServiceImpl.deleteAndReturn`

Reply:
Makes sense, updated.